challenge.md

```diff
@@ -6,19 +6,28 @@ Write your answers inside this file, where it's indicated by the comments.
 
 1. We just forked a repo on GitHub.com and want to start working on it locally. What command do we use to do that?
 
+git clone + SSH URL
+
 2. OK, we just wrote some code. What command could we use to see a list of all the changes that have been made since the last commit?
 
+git log
+
 
 3. We want to see more info about a specific commit, specifically who created the commit and when it was done. What command would one use?
 
-4. We are tasked with working on a new feature. And want to seperate our work from others in the project. What you we do? What git command would one use?
+git checkout + file name
+
+4. We are tasked with working on a new feature. And want to seperate our work from others in the project. What do we do? What git command would one use?
 
+git checkout -b to create a new branch
 
-5. I want to add a file to the staging area. What git commit would I use?
+5. I want to add a file to the staging area. What git command would I use?
 
+git add + file
 
 6. I've been working on a branch, add-directories, to implement a feature. The feature is done and you need to add your changes into the master branch. What git command should I use?
 
+git merge master
```

Comment:
Not quite. `git merge {branch}` will take the given branch and merge it into _your current location in the repo_. In this case, we want to merge the changes made on `add-directories` into `master`, so we'll need to move to the `master` branch (`git checkout master`) if we haven't already, and then run `git merge add-directories`.